### ISSUE TITLE
The button touch is not natural.

### ISSUE THREAD

**Issue by code918** [jacklam718/react-native-popup-dialog#260]
I received a report from a customer that the touch requires multiple presses to work.
Before that, I wasn't aware of it, but my tests showed that I needed to touch the button very carefully to make it work.
In particular, this is what happens on Android.
This is related to the touch gesture.
Pressing the button and moving it slightly does not cause the push action.
Very sensitive.
There seems to be a solution to change the touchable package as a way to solve this problem.
It is resolved by changing the package from 'react-native' to 'react-native-gesture-handler'.
But I didn't have the ability to modify it, so I couldn't change it and gave up using this library.
Be careful when anyone uses this library.

**Comment by wandizer** [comment]
Yes, I have the exact same issue on Android, not only the buttons inside, but also theh onTouchOutside doesn't seem to be registering for 1 sec or 2 when the modal shows up. (On Android only).
